modules/vault-secrets.nix

```diff
@@ -135,48 +141,37 @@ in
 {
   options = {
     vault-secrets = {
-      vaultPathPrefix = mkOption {
+      vaultPrefix = mkOption {
         type = with types; str;
         default = "kv";
         description = ''
           Base Vault KV path to prepend to all KV paths, including the mount point.
         '';
       };
 
-      vaultAddress = mkOption {
+      outPrefix = mkOption {
         type = with types; str;
-        default = "https://127.0.0.1:8200";
+        default = "/run/secrets";
         description = ''
-          The address of the Vault server, passed via <literal>VAULT_ADDR</literal> environment variable.
-      '';
-      };
+          Base path to output secrets. The path will be created, owned by root, and chmod 700.
 
-      namespace = mkOption {
-        type = with types; str;
-        default = "services";
-        description = ''
-          Base Vault KV path to prepend to all KV paths, under
-          <literal>vaultPathPrefix</literal>. Default for all secrets defined in
-          the module.
+          Should probably be on tmpfs to avoid leaking secrets.
         '';
       };
 
       approlePrefix = mkOption {
         type = with types; nullOr str;
         default = null;
         description = ''
-          Prepended to the secret name for resolving the default environmentFile path.
```

Comment:
Why?

Reply:
Because we have things like `pegasus-banana-thingy-secret-for-foo` approles, sincce they are not namespaced. This setting allows you to set it to `pegasus-banana` and then use normal secret names like `thingy-secret-for-foo`.

Reply:
I was asking why you have removed the description here.

Reply:
Oh, I think I intended to rewrite it and forgot.